### PR TITLE
inital release - Swiss Public Connections

### PR DESCRIPTION
**Pull request by OliverMannhart**
## Description

This extension will use the API https://transport.opendata.ch/ for public transit connections. The user can enter the desired connection and time to receive the next scheduled connections.

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/swiss-ov/src/search-connection.tsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+      actions={
+        <ActionPanel>
+          <Action.SubmitForm onSubmit={handleSubmit} />
+        </ActionPanel>
+      }
+    >
+      <Form.Description text={"Swiss Public Transport Connection"} />
+      <Form.TextField id="from" title="From" placeholder="From" storeValue />
+      <Form.TextField id="to" title="To" placeholder="To" storeValue />
+      <Form.Checkbox id="isArrival" title="Ankunft" label="" storeValue />
```
Comment (pernielsentikaer) [inline extensions/swiss-ov/src/search-connection.tsx:84]: Could we keep this english

File: extensions/swiss-ov/src/search-connection.tsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    <Form
+      actions={
+        <ActionPanel>
+          <Action.SubmitForm onSubmit={handleSubmit} />
+        </ActionPanel>
+      }
+    >
+      <Form.Description text={"Swiss Public Transport Connection"} />
+      <Form.TextField id="from" title="From" placeholder="From" storeValue />
+      <Form.TextField id="to" title="To" placeholder="To" storeValue />
```
Comment (pernielsentikaer) [inline extensions/swiss-ov/src/search-connection.tsx:83]: Is there a way to use a dropdown with values instead? Pure text might be hard?

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/swiss-ov/src/search-connection.tsx (hunk 1 above)
Comment (OliverMannhart) [reply extensions/swiss-ov/src/search-connection.tsx:84]: Yes sure, I will change that👍

File: extensions/swiss-ov/src/search-connection.tsx (hunk 2 above)
Comment (OliverMannhart) [reply extensions/swiss-ov/src/search-connection.tsx:83]: You don't need to enter the full name of the station. ZIP codes will work too. The API will try to find the best match.

I think people will use it to get home from the office (on the road, there are better mobile solutions) and therefore will use the same from-to entries. To help with that, the extension will load the last entries for "from" and "to."

But of course, I see your point. Maybe in the next release, I could add the possibility to save favorite stations.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Comment by OliverMannhart** [comment]
Nice, thanks👍

**Pull request merged**